crates/agglayer-node/src/epoch_synchronizer.rs

```diff
@@ -21,7 +25,24 @@ impl EpochSynchronizer {
         EpochsStore::PerEpochStore: PerEpochReader + PerEpochWriter,
     {
         while opened_epoch.get_epoch_number() < current_epoch_number {
-            opened_epoch.start_packing()?;
+            match opened_epoch.start_packing() {
+                Err(StorageError::AlreadyPacked(_)) => {
+                    info!(
+                        "Epoch {} already packed, continue",
+                        opened_epoch.get_epoch_number()
+                    );
+                }
+                Err(error) => {
+                    error!(
+                        "Error starting packing for epoch {}: {:?}",
+                        opened_epoch.get_epoch_number(),
+                        error
+                    );
+
+                    return Err(error)?;
```

Comment:
The question mark operator is a bit wierd.

Would one of the following work?
```suggestion
                    return Err(error);
```
```suggestion
                    return Err(error.into());
```